pokemongo_bot/cell_workers/incubate_eggs_worker.py

```diff
@@ -2,65 +2,110 @@
 from pokemongo_bot.human_behaviour import sleep
```

Comment:
should've caught those when this ugly mess got merged the first time ;) I'll remove them